src/main/java/com/woowacamp/soolsool/global/exception/ErrorCode.java

```diff
@@ -0,0 +1,16 @@
+package com.woowacamp.soolsool.global.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+
+@Getter
+@RequiredArgsConstructor
+public enum ErrorCode {
+    DEFAULT_ERROR(400, "D101", "기본 예외입니다."),
+    ;
```

Comment:
클래스 시작부는 개행해 주시면 좋을 것 같아여!!
```suggestion

    DEFAULT_ERROR(400, "D101", "기본 예외입니다."),
    ;
```

Reply:
ok